### ISSUE TITLE
make4ht issue with longtable, there are two additionl empty rows generated which lead to to long vertical lines

### ISSUE THREAD

**Issue by cjohn001** [michal-h21/make4ht#110]
Hello,
I am trying to create html mit make4ht And I am using the longtable code below.
As can be seen below, the generated html gets two empty bottom rows (id=TBL-3-7- and id=TBL-3-8-) to much which leads to the vertical lines of the table being to long. Is there a workaround to the issue? Thanks for your help!

Seems I have to set the padding:0; on the two classes to get rid of the extra height.

```
\begin{longtable}[!ht]{|>{\centering\arraybackslash}p{0.205\linewidth}|>{\centering\arraybackslash}p{0.355\linewidth}|>{\raggedright\arraybackslash}p{0.44\linewidth}|}
	\hline \multicolumn{1}{|c|}{\textbf{N1}} & \multicolumn{1}{c|}{\textbf{Function}} & \multicolumn{1}{c|}{\textbf{ Source}} \tabularnewline  \hline
	\endfirsthead
	\hline \multicolumn{1}{|c|}{\textbf{N1}} & \multicolumn{1}{c|}{\textbf{Function}} & \multicolumn{1}{c|}{\textbf{ Source}} \tabularnewline  \hline
	\endhead
	\hline
	\endfoot

	line1                                    & f1                                     & t1                                 			\tabularnewline \hline
	line2                                    & f2                                     & t2
\end{longtable}
```

<img width="360" alt="Bildschirm­foto 2023-03-06 um 20 45 05" src="https://user-images.githubusercontent.com/43253483/223214722-d27edb5b-d61e-4cae-956a-7e340e6a54bc.png">
<img width="502" alt="Bildschirm­foto 2023-03-06 um 20 45 43" src="https://user-images.githubusercontent.com/43253483/223214750-cd08d7cb-34db-4424-bcf6-dfcf978e244e.png">

**Comment by michal-h21** [comment]
Thanks for the report. It should be fixed by d6aa83a.

**Comment by cjohn001** [comment]
@michal-h21 : Thank you very much. I have one question. How can I install the package? I am on a Mac and have MacTex distribution installed
